network/src/constants.rs

```diff
@@ -8,7 +8,7 @@
 // with your use-case. If you do change a value, please add a comment linking to the PR which
 // advocated the change.
 /// The timeout for any inbound RPC call before it's cut off
-pub const INBOUND_RPC_TIMEOUT_MS: u64 = 10_000;
+pub const INBOUND_RPC_TIMEOUT_MS: u64 = 60_000;
```

Comment:
An RPC call shouldn't take 60 seconds.  Usually a connection timeout will be anywhere from 1-10 depending on the situation.

What RPC call is taking that long, and how can we improve the performance of it?

Reply:
Aah, I probably didn't make the PR description clear enough. My motivation for this PR was that it's weird to have an inbound RPC timeout of 10 seconds. For example, imagine I make an outbound RPC call with a 12 second timeout. Will it have a 12 second timeout in practice? No, because the receiver will kill it if the application doesn't respond in 10 seconds. I thought this was too aggressive. 

I get that we want something here (e.g., for garbage collection if there's a bug in the application layer), but 10 seconds seems too short to me given that we're already seeing some RPC calls take longer than 10 seconds in DevNet, today.

Reply:
My recommendation is that it seems weird to have an inbound timeout except for in cases of garbage collection -- but a timeout is really ... awkward in that the underlying job is either still executing and hasn't returned or there was some crazy error that didn't kill the node but killed the worker. I'd be in favor of a much higher timeout here. I'd also be curious to see what we can do in forge as well as just setting a single node in devnet this week before pulling the trigger.

Thoughts? @JoshLind , @gregnazario , @sherry-x

Reply:
An inbound timeout is only useful because there's really not much of a connection timeout.  There should be at least one

Reply:
Well, there is ... isn't there? Health checker does time out after a couple minutes, no?

Reply:
I mean, yes but that means that you're waiting minutes on something that might not run and it can eat up resources that are idle